### PR TITLE
Switch to latest VS Code test package.

### PR DESCRIPTION
**Pull request by philliphoff**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1079,7 +1080,6 @@
         "vinyl-buffer": "^1.0.1",
         "vinyl-source-stream": "^2.0.0",
         "vsce": "^1.87.0",
-        "vscode-test": "^1.5.2",
```
Comment (bwateratmsft) [inline package.json:1082]: Evidently something is still depending on `vscode-test`; do we know what?

**Review by bwateratmsft — APPROVED** [review]

**Review by bwateratmsft — APPROVED** [review]

**Review by bwateratmsft — APPROVED** [review]

**Comment by alexweininger** [comment]
Tests are currently failing due to the bug outlined here https://github.com/microsoft/vscode-azure-account/pull/603

File: test/runTest.ts [hunk 2]
```diff
@@ -3,11 +3,28 @@
...
+        const vscodeExecutablePath = await downloadAndUnzipVSCode('1.66.0');
+        const [cli, ...args] = resolveCliArgsFromVSCodeExecutablePath(vscodeExecutablePath);
+
+        cp.spawnSync(
+            cli,
+            [
+                ...args,
+                '--install-extension', 'ms-vscode.azure-account',
+                '--install-extension', 'ms-azuretools.vscode-azureresourcegroups',
+                '--install-extension', 'ms-python.python',
```
Comment (alexweininger) [inline test/runTest.ts:21]: FYI the gulp tasks for installing extensions we use do not actually install the extensions. They just place a fake package.json to fool VS Code into thinking the extensions are installed.

Comment (bwateratmsft) [reply test/runTest.ts:21]: Eventually we're going to have to just get rid of Gulp.

**Comment by alexweininger** [comment]
I did this in #3275

**Pull request closed without merging**